backend/user/repository/postgresql.go

```diff
@@ -49,7 +49,7 @@ func (p *postgresqlUserRepo) GetByID(ctx context.Context, id int64) (*swagger.Us
 func (p *postgresqlUserRepo) GetAllUser(ctx context.Context) ([]swagger.UserDataShort, error) {
 	sqlStatement := `
 		SELECT 	email, user_id, name, authority, status
-		FROM user_data;
+		FROM user_data WHERE user_id != 1;
 	`
```

Comment:
i think it is better that userdata can order by id

Reply:
maybe I can 
WHERE authority != 'admin' ?

Reply:
fix